app/services/return-requirements/submit-check.service.js

```diff
@@ -28,6 +29,8 @@ async function go (sessionId) {
 
   await _validateLicence(session.licence.id)
 
+  await PersistReturnRequirementsService.go(session)
```

Comment:
Do we need to prevent a user from approving if there are no return requirements ? (This has now been asked on the ticket) 

Should we create a requirement version on it's own if there are no returnRequirements. I can not see where someone can select not required for return ?